src/nodemcuWeatherAtHome/nodemcuWeatherAtHome.ino

```diff
@@ -58,7 +126,31 @@ void renderDisplay() {
   display.clearDisplay();
   display.dim(true);
 
-  display.drawLine(0, 14, SCREEN_WIDTH, 14, WHITE);
+  display.fillRect(0,0,128,18,WHITE); // x, y, w, z | x,y Startposition von oben links w breite horzontal, z höhe vertikal
+  
+  if (RSSI > -65) {
```

Comment:
Switch case with fall through.

Reply:
Komparatoren werden in C switch case nicht unterstützt.
https://stackoverflow.com/questions/20972297/larger-than-and-less-than-in-c-switch-statement